src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt

```diff
@@ -143,18 +144,48 @@ class ApplicationDataServiceOpenShift(
             )
         }
 
-        val deployDetails = createDeployDetails(dc)
+        fun getReplicationController(rcVersion: Long): ReplicationController? {
+            val rcName = "${dc.metadata.name}-$rcVersion"
+            return openshiftService.rc(namespace, rcName)
+        }
+
+        fun getRunningReplicationController(): ReplicationController? {
+            if (getReplicationController(dc.status.latestVersion)?.isRunning() == true) {
```

Comment:
Kan du assigne getReplicationController til en variabel her så du slipper å hente den to ganger?

evt gjøre getReplicationControlleer(...).takeIf

Reply:
Jepp, Espen nevnte det i går. Pushet nettopp en foreslått endring på dette.